docs/howto/config.rst

```diff
@@ -881,6 +917,19 @@ Example::
         },
     },
 
+only_use_keys_in_metadata
+"""""""""""""""""""""""""
+
+If set to False, the certificate contained in a SAML message will be used for
+signature verification.
+Default True.
```

Comment:
I think this configuration option should be removed in the future. We should not be using anything but the certificates we know about.